client/src/tests/e2e/contribuer.spec.ts

```diff
@@ -75,6 +76,15 @@ test.describe("Basic form submission", () => {
       label: UPDATE_FREQUENCY_LABELS.daily,
     });
 
+    // "Mots clés" section
+
+    const tags = page.locator("form [name=tags]");
+
+    await tags.fill(tagName);
+    const selectedTag = page.locator("text=services");
+
+    expect(selectedTag).toBeDefined();
```

Comment:
Je ne pense pas que ça fasse ce qu'on pense. À mon avis c'est une assertion générique `!== undefined`, or `page.locator()` renvoie un `Locator` qui est toujours défini.

Le but est de vérifier que le tag est bien affiché à l'écran, c'est ça ?

Peut-être qu'on peut trouver qqc qui dise "expect <element> to contain <text>", l'élément pourrait être la liste de tags.

Reply:
J'ai trouvé ceci : https://stackoverflow.com/questions/64784781/how-to-check-if-an-element-exists-on-the-page-in-playwright-js

Reply:
Solution proposée : 

```
   const selectedTag = await page.$$("text=services");

    expect(selectedTag).toHaveLength(1)
```